acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -298,11 +310,16 @@ private void printOutput(final BesuNode node, final Process process) {
         new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))) {
       String line = in.readLine();
       while (line != null) {
-        PROCESS_LOG.info("{}: {}", node.getName(), line);
+        // would be nice to pass up the log level of the incoming log line
+        PROCESS_LOG.info(line);
```

Comment:
Aren't you loosing the name of the node here (that name being assigned by the test being executed)?

Reply:
Yes, it is removed here and instead added in log4j2.xml from the ThreadContext.

Reply:
fair enough